nni/experiment/config/convert.py

```diff
@@ -86,24 +86,84 @@ def to_v1_yaml(config: ExperimentConfig, skip_nnictl: bool = False) -> Dict[str,
                 'maxTrialNumPerGpu': machine['maxTrialNumPerGpu'],
                 'useActiveGpu': machine['useActiveGpu'],
                 'preCommand': machine['trialPrepareCommand']
-            }
+            })
 
     elif ts['platform'] == 'pai':
-        data['trial']['cpuNum'] = ts['trialCpuNumber']
-        data['trial']['memoryMB'] = util.parse_size(ts['trialMemorySize'])
         data['trial']['image'] = ts['docker_image']
+        data['trial']['nniManagerNFSMountPath'] = ts['local_storage_mount_point']
+        data['trial']['containerNFSMountPath'] = ts['container_storage_mount_point']
         data['paiConfig'] = {
             'userName': ts['username'],
             'token': ts['token'],
             'host': 'https://' + ts['host'],
```

Comment:
The host does not contains prefix? how about `http://`?

Reply:
Do you mean there are cases that OpenPAI cluster supports only HTTP and not HTTPS?